pytorch_lightning/loops/base.py

```diff
@@ -99,6 +99,51 @@ def connect(self, **kwargs: "Loop") -> None:
         Linked loops should form a tree.
         """
 
+    def replace(self, **loops: Union["Loop", Type["Loop"]]) -> None:
+        """Optionally replace one or multiple of this loop's sub-loops.
+
+        This methods takes care of instantiating the class (if necessary) with all existing arguments, connecting all
+        sub-loops of the old loop to the new instance, setting the ``Trainer`` reference, and conneting the new loop to
+        the parent.
+
+        Args:
+            **loops: A ``Loop`` subclass or instance. The name used should match the loop attribute name you want to
```

Comment:
`A` loop?

Reply:
What do you suggest exactly? Note these are `**kwargs` so you pass them individually.

Reply:
A sequence of ``Loop``(s), maybe?